collections/_release_4_5/entry-systems-xr-openxr-render-models.md

```diff
@@ -0,0 +1,24 @@
+---
+type: entry
+section: systems
+subsection: XR
+rank: 2
+importance: 3
+anchor: openxr-render-models
+title: OpenXR render models
+text: |
+  Godot added support for the `XR_EXT_render_model` and `XR_EXT_interaction_render_model` extensions introduced in OpenXR.
+
+  These extensions, when enabled and supported, give an application access to fully animated models of the controllers currently held by the users, including tracking data for these controllers. This means Meta Quest 3S and Pico 4 Ultra users will see their respective controllers in their hands without extra efforts from the developer.
+
+  In order to use this feature in your game, you need to add <code class="highlight"><span class="enginetype">OpenXRRenderModels</span></code> as a child node of the <code class="highlight"><span class="enginetype">XROrigin3D</span></code> node to make the system work. For more information, [read the docs](https://docs.godotengine.org/en/4.5/tutorials/xr/openxr_render_models.html).
```

Comment:
```suggestion
  In order to use this feature in your game, you need to add <code class="highlight"><span class="enginetype">OpenXRRenderModels</span></code> as a child node of the <code class="highlight"><span class="enginetype">XROrigin3D</span></code> node. For more information, [please see the docs](https://docs.godotengine.org/en/4.5/tutorials/xr/openxr_render_models.html).
```
In English this direct phrasing is strangely perceived as aggressive

Also repetition of "In order to use this feature" and "to make the system work"